test/functional/cmdLineTests/classLoaderTest/src/org/openj9/test/illegalAccessError/IllegalAccessErrorTest.java

```diff
@@ -0,0 +1,158 @@
+ * is available at https://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * This Source Code may also be made available under the following
+ * Secondary Licenses when the conditions for such availability set
+ * forth in the Eclipse Public License, v. 2.0 are satisfied: GNU
+ * General Public License, version 2 with the GNU Classpath
+ * Exception [1] and GNU General Public License, version 2 with the
+ * OpenJDK Assembly Exception [2].
+ *
+ * [1] https://www.gnu.org/software/classpath/license.html
+ * [2] https://openjdk.org/legal/assembly-exception.html
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0-only WITH Classpath-exception-2.0 OR GPL-2.0-only WITH OpenJDK-assembly-exception-1.0
+ */
+
+package org.openj9.test.illegalAccessError;
+
+import java.lang.reflect.InvocationTargetException;
+import java.lang.reflect.Method;
+import java.io.InputStream;
+import java.io.IOException;
+import java.net.URL;
+import java.net.URLClassLoader;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+import org.openj9.test.util.VersionCheck;
+
+public class IllegalAccessErrorTest {
+
+	public static void main(String[] args) throws Exception {
+		boolean result;
+		URLClassLoader ucl = new URLClassLoader(new URL[] {});
+		Method method = ClassLoader.class.getDeclaredMethod("defineClass", String.class, byte[].class, int.class, int.class);
+		method.setAccessible(true);
+		try (InputStream in = ucl.getResourceAsStream("org/openj9/test/illegalAccessError/ExtendsDefaultVisibility.class")) {
+			byte[] bytes = new byte[in.available()];
+			in.read(bytes);
+			method.invoke(ucl, "org.openj9.test.illegalAccessError.ExtendsDefaultVisibility", bytes, 0, bytes.length);
+			System.out.println("defineClass for ExtendsDefaultVisibility should throw IllegalAccessError but didn't, test failed");
+			result = false;
+		} catch (InvocationTargetException e) {
+			String causeString = e.getCause().toString();
+			Pattern pattern = Pattern.compile(
+					"java.lang.IllegalAccessError:"
+						+ " class org/openj9/test/illegalAccessError/ExtendsDefaultVisibility"
+						+ " \\(in unnamed module 0x0{1,16} from loader java/net/URLClassLoader\\)"
+						+ " cannot access its superclass"
+						+ " org/openj9/test/illegalAccessError/DefaultVisibility"
+						+ " \\(in unnamed module 0x0{1,16} from loader jdk/internal/loader/ClassLoaders\\$AppClassLoader\\)");
+			Matcher matcher = pattern.matcher(causeString);
+			System.out.println("causeString = " + causeString);
+			if (((VersionCheck.major() == 8) && causeString.contains(
+					"java.lang.IllegalAccessError:"
+						+ " class org/openj9/test/illegalAccessError/ExtendsDefaultVisibility"
+						+ " (from loader java/net/URLClassLoader) cannot access its superclass"
+						+ " org/openj9/test/illegalAccessError/DefaultVisibility"
+						+ " (from loader sun/misc/Launcher$AppClassLoader)"))
+					|| ((VersionCheck.major() >= 11) && matcher.find())) {
```

Comment:
This line is indented one more tab than I think necessary (it does, but shouldn't, line up with line 61).
The final `) {` should go on a line by itself:
```java
				|| ((VersionCheck.major() >= 11) && matcher.find())
			) {
```
Same comment for lines 102 and 138.

Reply:
Updated